_docs/_latest/develop/test/integration-testing/run-test-suite.md

```diff
@@ -0,0 +1,148 @@
+order: 045
+---
+
+# {{ page.title }}
+
+This page describes how to run the integration test suite for your Forseti
+contributions on your local dev environment. 
+
+## **Setting up development environment**
+
+- Create a new project in your organization.
+- Terraform uses an IAM Service Account to deploy and configure resources on 
+behalf of the user. You can create the Service account, and grant the necessary 
+roles to the Service Account manually or by running a helper script. 
+
+### **Creating Service Account by running the helper script**
+
+In order to execute this script, you must have an account with the following 
+list of permissions: 
+* resourcemanager.organizations.list
+* resourcemanager.projects.list
+* billing.accounts.list
+* iam.serviceAccounts.create
+* iam.serviceAccountKeys.create
+* resourcemanager.organizations.setIamPolicy
+* resourcemanager.projects.setIamPolicy
+* serviceusage.services.enable on the project
+* servicemanagement.services.bind on following services:
+  * cloudresourcemanager.googleapis.com
+  * cloudbilling.googleapis.com
+  * iam.googleapis.com
+  * admin.googleapis.com
+  * appengine.googleapis.com
+* billing.accounts.getIamPolicy on a billing account.
+* billing.accounts.setIamPolicy on a billing account.
+- `Billing Account Administrator` role should be granted to the admin account to 
+be able to create the Service Account using the helper script. You can revoke 
+this role after the service account is created.
+- To run the [helper script](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/helpers/setup-sa.sh), 
+clone the [Terraform Google Project factory repository](https://github.com/terraform-google-modules/terraform-google-project-factory)
+and run the following command:
+
+`./helpers/setup-sa.sh <ORGANIZATION_ID> <PROJECT_NAME> <BILLING_ACCOUNT>`
+
+### **Creating Service Account and granting roles manually**
+
+Alternatively, you can grant the following roles and enable the APIs manually.
+
+On the organization:
+
+* roles/resourcemanager.organizationAdmin
+* roles/iam.securityReviewer
+* roles/resourcemanager.folderViewer
+* roles/resourcemanager.organizationViewer
+* roles/resourcemanager.projectCreator
+* roles/billing.user
+
+On the project:
+
+* roles/owner
+* roles/compute.instanceAdmin
+* roles/compute.networkViewer
+* roles/compute.securityAdmin
+* roles/iam.serviceAccountAdmin
+* roles/serviceusage.serviceUsageAdmin
+* roles/iam.serviceAccountUser
+* roles/storage.admin
+* roles/cloudsql.admin
+
+Enable the following APIs on the Forseti project:
+
+* cloudresourcemanager.googleapis.com
+* compute.googleapis.com
+* serviceusage.googleapis.com
+
+On the host project when using shared VPC
+
+* roles/compute.securityAdmin
+* roles/compute.networkAdmin
+* roles/browser
+* roles/resourcemanager.projectIamAdmin
+
+On the organization when using shared VPC
+
+* roles/compute.securityAdmin
+* roles/compute.networkAdmin
+* roles/compute.xpnAdmin
+
```

Comment:
There is some more manual work that users will need to do to run some of the tests in a their own environment. Examples:

- The Inventory performance test uses mock data files that are stored in a GCS bucket of the test project. Users will need a way to get these files or they can comment out this test. If they do have access to these files, then the  `TF_VAR_inventory_performance_cai_dump_paths` environment variable below will need to be set to the paths of these files, for example: `['gs://{BUCKET}/inventory_performance/mock_cai_iam_policy.dump','gs://{BUCKET}/inventory_performance/mock_cai_resource.dump']`
- For the email notification test, an encrypted file is stored in the repo that is used to check that the emails are received. This test requires that a KMS key is created and used to encrypt/decrypt these credentials. The file in the repo is encrypted for the Forseti test org where these tests are run. In order for others to run this test, they will need to have a Gmail account and store the encrypted credentials as [gmail_token.pickle.enc](https://github.com/forseti-security/forseti-security/blob/master/integration_tests/tests/forseti/scripts/gmail_token.pickle.enc). Then the `TF_VAR_kms_key` and `TF_VAR_kms_keyring` env vars will need to be set to the KMS keyring and key that was used to encrypt this file.

Reply:
Thanks! Added a note to reach out to the Forseti team to run inventory performance tests and email notification test as the team hasn't decided how to grant access to mock files to the users and requires manual work.

On a side note, #3738 has been opened to add instructions to add pytest.